src/features/upload-file/types/property-custom.type.ts

```diff
@@ -7,14 +7,16 @@ import { MimeType } from './mime-types.type'
 type PropertyCustom = {
   filePathProperty: string,
   fileProperty: string,
+  filesToDeleteProperty: string,
   keyProperty: string,
   fileNameProperty?: string,
   mimeTypeProperty?: string,
   bucketProperty?: string,
   defaultBucket: string,
-  mimeTypes?: Array<MimeType>,
+  mimeTypes?: Array<MimeType | string>,
```

Comment:
this one caused problems when I used it outside this package - so I simply added string so tsc won complain when user gave some mimetype i don't have in list